src/models/hit-types/item-hit.ts

```diff
@@ -0,0 +1,296 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Memoize } from 'typescript-memoize';
+import { BooleanField } from '../metadata-fields/field-types/boolean';
+import { DateField } from '../metadata-fields/field-types/date';
+import { MediaTypeField } from '../metadata-fields/field-types/mediatype';
+import { NumberField } from '../metadata-fields/field-types/number';
+import { StringField } from '../metadata-fields/field-types/string';
+
+/**
+ * A model that describes an item hit from a Metadata Search via the PPS endpoint.
+ *
+ * The fields in here are cast to their respective field types. See `metadata-fields/field-type`.
+ *
+ * Add additional fields as needed.
+ *
+ * @export
+ * @class ItemHit
+ */
+export class ItemHit {
+  /**
+   * This is the raw hit response; useful for inspecting the raw data
+   * returned from the server.
+   */
+  rawMetadata?: Record<string, any>;
+
+  constructor(json: Record<string, any>) {
+    this.rawMetadata = json;
+  }
+
+  /**
+   * The item identifier.
+   *
+   * _Note_: This is a plain string instead of a `MetadataField` since it's
+   * the primary key of the item.
+   */
+  get identifier(): string | undefined {
```

Comment:
```suggestion
  get identifier(): Metadata.identifier {
```

Reply:
Something like this? (in current commit)

```js
  get identifier(): typeof Metadata.prototype.identifier {
    return this.rawMetadata?.fields?.identifier;
  }

  /**
   * May be a superset of metadata collection field.
   * Multivalued.
   */
  @Memoize() get collection(): typeof Metadata.prototype.collection {
    return this.rawMetadata?.fields?.collection
      ? new StringField(this.rawMetadata.fields.collection)
      : undefined;
  }

// ...
```